docs/components/input.md

```diff
@@ -33,7 +33,7 @@ sl-input label="What is your name?"
 ```
 
 ```jsx react
-import { SlIcon, SlInput } from '@teamshares/shoelace/dist/react';
```

Comment:
Why are we removing this import?

Reply:
cause when i was copying it over, it was not used within the snippet

Reply:
We're not really using the React examples in our apps anyway, but I've been avoiding modifying them because a) it creates more differences from upstream and b) we may someday use the React versions, possibly in our Next.js apps. So I'd leave them alone unless it's actually a change you mean to make